src/pages/home/report/ReportActionItemSingle.tsx

```diff
@@ -126,11 +128,12 @@ function ReportActionItemSingle({
     } else {
         secondaryAvatar = {name: '', source: '', type: 'avatar'};
     }
+
     const icon = {
-        source: avatarSource,
+        source: avatarSource ?? FallbackAvatar,
         type: isWorkspaceActor ? CONST.ICON_TYPE_WORKSPACE : CONST.ICON_TYPE_AVATAR,
         name: primaryDisplayName ?? '',
-        id: isWorkspaceActor ? '' : actorAccountID,
+        id: isWorkspaceActor ? undefined : avatarAccountId,
```

Comment:
```suggestion
        id: avatarAccountId,
```
Update the `avatarAccountId` in the if condition `isWorkspaceActor`

Reply:
what I wanted to avoid is passing the `avatarAccountId` because I thought that for a "workspaceActor" there is some other id in this variable. 
But I guess you are right so I fixed this.